pages/common/conda.md

```diff
@@ -0,0 +1,35 @@
+# conda
+
+> Package, dependency and environment management for any programming language.
+
+- Create a new environment, installing named packages (eg. python=2.7):
```

Comment:
I was thinking about making the change in the token itself. That'd be more directly understandable. Maybe an example with two packages would make it clear how to pass several packages to this tool.